app/src/ingestion/la_policy/scrape/requirements.txt

```diff
@@ -0,0 +1,2 @@
+install-playwright
+playwright
```

Comment:
Needed to install `scrapy` for this env when running `python scrape_la_policy.py`, as I did not already have it.

```suggestion
playwright
scrapy
```

Reply:
Thanks!